### ISSUE TITLE
Estimated time calculation in the progress text [Feature Request]

### ISSUE THREAD

**Issue by naryal2580** [ffuf/ffuf#270]
Request for a estimated time calculation along with Duration within progress text while ffuf is running, this would be really helpful while fuzzing a huge list. Thanks 😃

**Comment by joohoi** [comment]
I agree that it'd be really beneficial. Unfortunately the progress bar space is already really tight unless you are running terminal full screen. I'll add a feature request label so we can track this in the future if there's a possibility of adding it.
